tests/e2e/test_artifacts/generated_e2e_test_iree_artifacts.cmake

```diff
@@ -3739,6 +3867,8 @@ add_dependencies(iree-benchmark-import-models
   ${PACKAGE_NAME}_iree-imported-model-213fe9a8738a01f2b02b6f0614a40a31c83a2603ca3e3ae0aeab8090fedbe3a0
   ${PACKAGE_NAME}_iree-imported-model-d4a10c6d3e8a11d808baf398822ea8b61be07673517ff9be30fbe199b7fdd960
   ${PACKAGE_NAME}_iree-imported-model-a122dabcac56c201a4c98d3474265f15adba14bff88353f421b1a11cadcdea1f
+  ${PACKAGE_NAME}_model-9a9515c7-cb68-4c34-b1d2-0e8c0a3620b8
+  ${PACKAGE_NAME}_model-340553d1-e6fe-41b6-b2c7-687c74ccec56
```

Comment:
Why are these names different?

Reply:
It's because they are the imported MLIR files instead of using importer to import PyTorch models every time we build the benchmark suite.

For the reason we directly use MLIR files: 

@mariecwhite discussed with me before. We think it's fine to directly use the imported MLIR for PyTorch benchmarks because:
1. The script to import PyTorch models requires lots of extra dependencies. We probably don't want to ask people to install them when building IREE benchmark suite
2. MLIR compatibility is the major reason we import TF/TFLite models in every build, but since we have no integration of `torch-mlir` in IREE repo, there is no benefit to importing PyTorch models to MLIR in every build.
3. I remembered Marie mentioned that the imported MLIR for PyTorch model is relatively stable. So it will only need to update the MLIR once a while when it breaks.

This is a mid-term solution before we can move the benchmark suite out of the IREE build system, then we can have extra dependencies to run torch-mlir for every build.

Reply:
Similar to comments on https://github.com/openxla/iree/pull/12017#discussion_r1093843034, if we're using artifacts using an unstable format they need to be stored with version information and the process for regenerating them needs to be clearly documented in a discoverable way.

Reply:
Version info is encoded in the GCS bucket hosting the mlir files: https://pantheon.corp.google.com/storage/browser/iree-model-artifacts/pytorch/torch_models_20230307.103_1678163233. `20230307.103` is the version of torch-mlir used. Inside the gcs bucket is a version_info.txt file that stores the output of `pip list`. It may be difficult to exactly reproduce the mlir files from scratch on a different machine since the packages may no longer be hosted. If we want exact reproducibility, we'll need to create images with the pip environment saved.

In terms of encoding this version info in the dashboards and/or database, I'll defer to @pzread.